src/scenes/list/List.js

```diff
@@ -76,24 +81,44 @@ class List extends React.Component {
     );
   }
 
+  renderScreen = () => {
+    const { startContainer, startText } = styles;
+    if (toDoStore.start) {
```

Comment:
This observable will not be watched. Only observables accessed inside `render` functions will be observed. Is this intended?

Reply:
This is actually not an error. I was wrong regarding how Mobx observes properties.